pkg/network/ovn_kubernetes.go

```diff
@@ -1472,7 +1472,8 @@ func hasSourceInMachineConfigStatus(machineConfigStatus mcfgv1.MachineConfigPool
 		sourceNames.Insert(machineConfig.Name)
 	}
 	for _, source := range machineConfigStatus.Configuration.Source {
-		if sourceNames.Has(source.Name) {
+		if sourceNames.Has(source.Name) &&
+			machineConfigStatus.MachineCount == machineConfigStatus.UpdatedMachineCount {
```

Comment:
I think this check should be done in `isIPsecMachineConfigActive`

Also, reading this code now it is confusing this method returns true if just one of the passed machineConfigs is present, vs all of them.

Reply:
yes @jcaamano , changed method name with a comment to reflect what it's intended to achieve, hope it's ok now.

Reply:
I meant to say move the check to the other method, not change this method's name.

Reply:
ok, that makes sense. done.